src/EllCrv/EllCrv.jl

```diff
@@ -778,8 +778,23 @@ end
 #
 ################################################################################
 
+function show(io::IO, ::MIME"text/plain", E::EllipticCurve)
+  io = pretty(io)
+  println(io, "Elliptic curve with equation")
+  print(io, Indent())
+  _print_equation(io, E)
+  print(io, Dedent())
+end
+
 function show(io::IO, E::EllipticCurve)
-  print(io, "Elliptic curve with equation\n")
+  io = pretty(io)
+  print(io, "Elliptic curve with equation ")
+  print(io, Indent())
```

Comment:
I think I got confused by this. What does this do without linebreak?

Reply:
Nothing (useful). I overlooked it. Thanks for pointing it out. Removed it.